tests/baselines/reference/shorthandPropertyAssignmentInES6Module.errors.txt

```diff
@@ -0,0 +1,23 @@
+tests/cases/compiler/test.ts(2,19): error TS2307: Cannot find module './foo2'.
+tests/cases/compiler/test.ts(6,1): error TS2304: Cannot find name 'console'.
+tests/cases/compiler/test.ts(7,1): error TS2304: Cannot find name 'console'.

```

Comment:
`console ` is irrelevant, However missing `./foo2` module is necessary to make sure that we emit the right thing no matter whether module is found or not.